src/Tizen.Applications.ComponentManager/Tizen.Applications/ComponentRunningContext.cs

```diff
@@ -0,0 +1,309 @@
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextGetInstanceId(_contextHandle, out instanceId);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the InstanceId of " + _componentId + ". err = " + err);
+                }
+
+                return instanceId;
+            }
+        }
+
+        /// <summary>
+        /// Gets the state of the component.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public ComponentState State
+        {
+            get
+            {
+                int state = 0;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextGetComponentState(_contextHandle, out state);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the State of " + _componentId + ". err = " + err);
+                }
+
+                return (ComponentState)state;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the component is terminated or not.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsTerminated
+        {
+            get
+            {
+                bool isTerminated = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextIsTerminated(_contextHandle, out isTerminated);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsTerminated of " + _componentId + ". err = " + err);
+                }
+
+                return isTerminated;
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the component is running as sub component of the group.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public bool IsSubComponent
+        {
+            get
+            {
+                bool isSubComponent = false;
+                Interop.ComponentManager.ErrorCode err = Interop.ComponentManager.ComponentContextIsSubComponent(_contextHandle, out isSubComponent);
+                if (err != Interop.ComponentManager.ErrorCode.None)
+                {
+                    Log.Warn(LogTag, "Failed to get the IsSubComponent of " + _componentId + ". err = " + err);
+                }
+
+                return isSubComponent;
+            }
+        }
+
+        /// <summary>
+        /// Resumes the running component.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when failed because of an invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of the system error.</exception>
+        /// <exception cref="OutOfMemoryException">Thrown when failed because of ouf of memory.</exception>
```

Comment:
Spelling, should be `out` instead of `ouf`.

Reply:
Done